app/mailers/teacher_mailer.rb

```diff
@@ -11,34 +11,38 @@ class TeacherMailer < ApplicationMailer
   def welcome_email(teacher)
     @teacher = teacher
     set_body
-    mail to: teacher.email_name,
+    set_recipients
```

Comment:
I am going to merge this, but `set_recipients` probably is best as a `before_action` since it's applied to all email templates.